src/migrations/2023080309391234-update-goal-on-approved-ar-value.js

```diff
@@ -0,0 +1,32 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await prepMigration(queryInterface, transaction, __filename);
```

Comment:
__filename needs to be resolved to a variable within this file before it's passed or it gets used as a null

Reply:
This has been updated.